cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -99,11 +99,17 @@ object ReporterCommand : CommandWithHelp() {
     private var repositoryConfigurationFile: File? = null
 
     override fun runCommand(jc: JCommander): Int {
-        require(!outputDir.exists()) {
-            "The output directory '${outputDir.absolutePath}' must not exist yet."
+        val absoluteOutputDir = outputDir.absoluteFile
+
+        val reports = reportFormats.associateWith { format ->
```

Comment:
Naming the variable `reporter` instead of `format` would be more consistent with the loop in line 129. It's a bit confusing that `reportFormats` actually contains `Reporter`s, maybe we should rename it to `reporters`. The question is what is more important, naming the variable like it's content or like the command line parameter.

Reply:
I've renamed both. I things it's fine to not match the command line paramater name to the variable name here, as we explicitly do a conversion via `ReporterConverter`. That is, while on the command line it is still *formats*, in the variable it is *reporters*.